src/Surfnet/StepupMiddleware/ApiBundle/Identity/Service/RaCandidateService.php

```diff
@@ -50,6 +50,26 @@ public function search(RaCandidateQuery $query)
         return $paginator;
     }
 
+    /**
+     * @param RaCandidateQuery $query
+     * @return array
+     */
+    public function getFilterOptions(RaCandidateQuery $query)
```

Comment:
Like my previous comment, please consider moving this logic to a helper class.

Reply:
See my comment above